apps/cli/.eslintrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  root: true,
+  extends: ["@webstudio-is/eslint-config-custom"],
```

Comment:
We have config in the root. This should not be necessary. You can tweak no-console with overrides in root config specifically for cli.